### PR TITLE
Check kbet lisi

### PR DESCRIPTION
**Pull request by mbuttner**

### CODE REVIEW

File: scIB/metrics/kbet.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def kBET(
         embed='X_pca',
         type_=None,
         hvg=False,
-        subsample=0.5,
+        subsample=0.5, #non-functional
```
Comment (mumichae) [inline scIB/metrics/kbet.py:81]: I'd suggest we remove that option completely, i.e also from `kBET_single` and `metrics`

**Review by mumichae — COMMENTED** [review]
Looking good so far. I think we could be more radical in removing never used parameters and comments. Are changes for the LISI functions also planned?

**Comment by mbuttner** [comment]
> Looking good so far. I think we could be more radical in removing never used parameters and comments. Are changes for the LISI functions also planned?

Thanks, I checked the LISI functions and did not find things that I would change.

**Comment by mumichae** [comment]
OK, then I would cleanup the code, which shouldn't change the score. That'll be subject to a new PR.

**Review by mumichae — APPROVED** [review]

**Comment by mbuttner** [comment]
Awesome, thank you!

**Pull request merged**